test/TestProjects/CadlPetStore/Generated/Pet.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace CadlPetStore
+{
+    public partial class Pet
+    {
+        public string Name { get; }
+
+        public string Tag { get; }
+
+        public int Age { get; }
```

Comment:
Could we put the constructor at the top in a class?

Reply:
@archerzz,  I think constructor is at the top in the design.

Reply:
> @archerzz, I think constructor is at the top in the design.

I did the research again, and I think you're correct. The recommended sequence is: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md